qstash/workflow/examples/allInOne.mdx

```diff
@@ -19,7 +19,7 @@ Our workflow will:
 ## Code Example
 
 ```typescript api/workflow/route.ts
-import { serve } from "@upstash/qstash/nextjs"
+import { serve } from "@upstash/workflow/nextjs"
```

Comment:
I noticed that you’ve updated `POST` to `{ POST }` in some examples, but we’re still using the former one here.

This applies to all the example projects.